docs/src/whatsnew/dev.rst

```diff
@@ -37,7 +37,7 @@ This document explains the changes made to Iris for this release
 🐛 Bugs Fixed
 =============
 
-#. N/A
+#. `@rsdavies`_ modified the CF complaint standard name for m01s00i023 :issue:`4566` 
```

Comment:
@rsdavies Awesome, thanks for this.

BTW the docs being built in CI are throwing the error:
```shell
ERROR: Unknown target name: "@rsdavies".
```
To fix this, simply add a reference to your `@rsdavies` target at the bottom of the `dev.rst` document e.g., add the following:
```
.. comment
    Whatsnew author names (@github name) in alphabetical order. Note that,
    core dev names are automatically included by the common_links.inc:

.. _@rsdavies: https://github.com/rsdavies
```

Reply:
@rsdavies Also, have you signed the CLA?

If not, here's the Google Form https://scitools.org.uk/cla/v4/form

Complete that to sign the CLA and we'll do the rest :+1:

Reply:
Thanks Bill, I've signed the CLA.

Reply:
And I've fixed the error in dev.rst :)